src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -298,15 +304,17 @@ foam.CLASS({
                       .callIf(self.config.searchMode === self.SearchMode.SIMPLE, function() {
                         this.tag(self.SimpleSearch, {
                           showCount: false,
-                          data$: self.searchPredicate$
+                          data$: self.searchPredicate$,
+                          searchValue: self.memento && self.memento.paramsObj && self.memento.paramsObj.search ? self.memento.paramsObj.search : null
```

Comment:
Is this the same as:
self.memento && self.memento.paramsObj && self.memento.paramsObj.search
?